.storybook/components/BestPractices.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ReactNode } from 'react'
+
+type BestPracticesProps = {
+  title?: string
+  description?: string
+  children: ReactNode
+}
+
+const BestPractices = ({ children }: BestPracticesProps) => {
```

Comment:
Hmm, what do you think about using `PropsWithChildren`?

```suggestion
import React from 'react'
import type { PropsWithChildren } from 'react'

type BestPracticesProps = {
  title?: string
  description?: string
}

const BestPractices = ({ children }: PropsWithChildren<BestPracticesProps>) => {
```